environment/create_google_config.py

```diff
@@ -0,0 +1,308 @@
+# Build a cloud billing API service
+billing = build('cloudbilling', 'v1', credentials=credentials)
+# Build a cloud resource manager API service
+crm = build('cloudresourcemanager', 'v1', credentials=credentials)
+# Create storage client
+storage_client = storage.Client()
+# Create Service management API service
+smgt = build('servicemanagement', 'v1', credentials=credentials)
+
+# Global variables
+project_name = ""
+bucket_name = ""
+
+#TODO: take out the error messages, info prompts, etc and put into another file (import them here)
+#TODO: consider making the configuration a class, so that a user could create multiple configurations
+#TODO: add nice error message for if user does keyboard interrupt (command + C) at any point in the script. Is it possible to make the errors specific to where they are in the process? post-created project, post-config created, etc?
+
+# The purpose of this script is to create a configuration file for Cromwell to run on Google Cloud with your local data.
+
+def main():
+	
+	# Ensure that the user has not run this script before, to avoid overwriting an existing configuration file
+	google_config_check()
+
+	print "\nHello and welcome! This script is for users who want to run their WDL scripts on Google Cloud\nusing the Cromwell engine. This script will walk you through the following setup steps:\n(1) Check that you have Google Cloud SDK installed,\n(2) Select an existing Google Project or create a new one,\n(3) Create a Google Bucket for the workflow outputs,\n(4) Create a Configuration file for running Cromwell,\n(5) Enable APIs for running workflows,\n(6) Test your configuration file by downloading Cromwell and running a \"Hello, World\" WDL.\n\nReady? Let's get started.\n"
+
+	# Ensure that gcloud SDK is installed, which is necessary to run the rest of the script. 
+	sdk_install_check()
+
+	# Select Google project (new or existing)
+	which_google_project()
+
+	# Create config
+	create_config()
+
+# Ensure that the user has not run this script before, to avoid overwriting an existing configuration file
+def google_config_check():
+	
+	# Check for .google_cromwell.config in ~/
+	home = os.path.expanduser("~")
+	existance = os.path.exists(home + '/.google_cromwell.config')
+	
+	# If there is a configuration file, exit
+	if existance:
+		print "\nYou already have a Cromwell configuration file. If you would like to clear this setup\nand create a new file, remove (or rename) the hidden file ~/.google_cromwell.config\n"
+		sys.exit("Exiting.")
+
+# Ensure that gcloud SDK is installed, which is necessary to run the rest of the script. 
+def sdk_install_check():
+		
+	# If gcloud SDK is not installed, install it.
+	if os.system('gcloud version') is not 0:
+		
+		# Ask if user wants to install the SDK
+		installSdk = raw_input('\nStep (1): You do not have Google Cloud SDK installed, which you need to run this script.\nDo you want to install gcloud SDK? (yes or no)').lower()
```

Comment:
we should change lines 70-76 into a reusable function for this purpose, I can help with that